### PR TITLE
support non-integer priorities

### PR DESCRIPTION
**Pull request by sirctseb**
priorities need not be integers

### CODE REVIEW

File: lib/src/firebase.dart [hunk 1]
```diff
@@ -358,7 +358,7 @@ class Firebase extends Query {
    * specific priority, and this function should be used when updating the
    * priority of existing data.
    */
-  Future setPriority(int priority) {
+  Future setPriority(priority) {
```
Comment (kevmoo) [inline lib/src/firebase.dart]: use `num` instead of leaving it blank?

File: lib/src/firebase.dart [hunk 2]
```diff
@@ -332,7 +332,7 @@ class Firebase extends Query {
    * Returns a Future which will complete when the data has been synchronized
    * with Firebase.
    */
-  Future setWithPriority(value, int priority) {
+  Future setWithPriority(value, priority) {
```
Comment (kevmoo) [inline lib/src/firebase.dart]: use `num` instead of leaving it blank?

File: lib/src/firebase.dart (hunk 1 above)
Comment (sirctseb) [inline lib/src/firebase.dart]: priorities can also be strings

**Comment by sirctseb** [comment]
priorities can also be strings. is dynamic better than unspecified?

**Comment by kevmoo** [comment]
Thanks! Committed as 539dbbe1c5af29bd87f8da8b6682d8e9d5ad95ac

**Pull request closed without merging**